qa/tasks/rgw.py

```diff
@@ -221,6 +218,8 @@ def ship_apache_configs(ctx, config, role_endpoints, on_client = None,
     finally:
         log.info('Removing apache config...')
         for client in clients_to_create_as:
+            cluster_name, daemon_type, client_id = teuthology.split_role(client)
+            client_with_cluster = '.'.join((cluster_name, daemon_type, client_id))
```

Comment:
what would you like to see logged here? just that it's getting the right value for `client_with_cluster`? here's a snippet from a [teuthology.log](http://qa-proxy.ceph.com/teuthology/cbodley-2017-05-16_10:58:37-rgw-master---basic-mira/1185156/teuthology.log) that shows this in action:
> INFO:tasks.rgw.client.0:Stopped
INFO:teuthology.orchestra.run.mira114:Running: 'rm -f /home/ubuntu/cephtest/rgw.opslog.ceph.client.0.sock'
INFO:tasks.rgw:Stopping apache...
INFO:tasks.rgw:Removing apache config...
INFO:teuthology.orchestra.run.mira114:Running: 'rm -f /home/ubuntu/cephtest/apache/apache.ceph.client.0.conf && rm -f /home/ubuntu/cephtest/apache/htdocs.ceph.client.0/rgw.fcgi'
INFO:tasks.rgw:Cleaning up apache directories...
INFO:teuthology.orchestra.run.mira114:Running: 'rm -rf /home/ubuntu/cephtest/apache/tmp.ceph.client.0 && rmdir /home/ubuntu/cephtest/apache/htdocs.ceph.client.0'
INFO:teuthology.orchestra.run.mira114:Running: 'rmdir /home/ubuntu/cephtest/apache'
DEBUG:teuthology.run_tasks:Unwinding manager ceph